server/src/services/personalAllDetails.service.ts

```
@@ -42,7 +43,32 @@ const getBiodata = async (id: string) => {
   return result;
 };
 
+const getPublicBiodata = async (req: CustomReq) => {
+  // Get page and limit from query params
+  const page = parseInt(req.query._page) || 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)